resources/assets/js/features/user/actions.js

```diff
@@ -51,4 +51,97 @@ export const doVerify = (email, token) => dispatch => {
             .then(response => dispatch(verifySuccess(response.data)))
             .catch(error => dispatch(verifyFailed(error.response.data)));
     }
+};
+
+
+export const loginSuccess = data => ({
+    type: actions.LOGIN_SUCCESS,
+    data
+});
+
+export const loginFormFailed = data => ({
+    type: actions.LOGIN_VERIFY_FAILED,
+    data
+});
+
+function processFailedLoginResponse(response) {
+    switch (response.status) {
+        case 401:
+            return {
+                type: actions.LOGIN_FAILED_NO_ACTIVATION,
+                response
+            };
+        case 404:
+            return {
+                type: actions.LOGIN_FAILED_NO_USER,
+                response
+            };
+        case 422:
+            return {
+                type: actions.LOGIN_FAILED_BAD_CREDENTIALS,
+                response
+            }
+        default:
+            return {
+                type: actions.LOGIN_FAILED,
+                response
+            }
+    }
+};
+
+export const doLogin = (credentials) => dispatch => {
+    const emailValid = RegisterValidator.email(credentials.email);
+    const passwordValid = RegisterValidator.password(credentials.password);
+
+    if (!emailValid.valid) {
+        return dispatch(loginFormFailed({ email: emailValid.error }));
+    }
+
+    if (!passwordValid.valid) {
+        return dispatch(loginFormFailed({ password: passwordValid.error }));
+    }
+
+    axios.post('/api/user/authorization', {
+        email: credentials.email,
+        password: credentials.password
+    })
+        .then(response => {
+            sessionStorage.setItem('jwt', response.data.token);
+            dispatch(loginSuccess(response.data))
+        })
+        .catch(error => {
+            sessionStorage.removeItem('jwt');
```

Comment:
all work with session should be in AuthManager/AuthService, it should know how to verify if session active or not